packages/expo-facebook/src/Facebook.ts

```diff
@@ -6,6 +6,7 @@ type FacebookLoginResult = {
   type: string;
   token?: string;
   expires?: number;
+  declinedPermissions: string[];
```

Comment:
```suggestion
  declinedPermissions?: string[];
```

Needs to be optional in the cancellation case.